internal/provisioner/plain/main.go

```diff
@@ -134,10 +134,18 @@ func main() {
 		setupLog.Error(err, "unable to parse bundle content server URL")
 		os.Exit(1)
 	}
-	bundleStorage := &storage.LocalDirectory{
+
+	localStorage := &storage.LocalDirectory{
 		RootDirectory: storageDirectory,
 		URL:           *storageURL,
 	}
+	httpLoader := storage.NewHTTP(
+		// TODO: Use a trusted CA to connect to bundle content URLs.
```

Comment:
Create tracking issue?

Reply:
This should happen automatically when this PR merges, right?